pkg/ddl/backfilling_scheduler.go

```diff
@@ -595,3 +603,76 @@ func (a *taskIDAllocator) alloc() int {
 	a.id++
 	return a.id
 }
+
+// estimateRowSize estimates the row size in bytes of a table.
+// This function tries to retrieve row size in following orders:
+//  1. statistic cache (require analyze table).
+//  2. region info's approximate key size / key number.
+func estimateRowSize(ctx context.Context, store kv.Storage, tbl table.Table) (sizeInBytes int) {
+	defer util.Recover(metrics.LabelDDL, "estimateRowSize", nil, false)
+	var gErr error
+	defer func() {
+		logutil.Logger(ctx).Info("estimate row size",
+			zap.Int64("tableID", tbl.Meta().ID), zap.Int("size", sizeInBytes), zap.Error(gErr))
+	}()
+	rowCount, avgRowLen, _, _ := cache.TableRowStatsCache.EstimateDataLength(tbl.Meta())
```

Comment:
statsCache only updated when we access infoschema.Table/Columns, may be read stats from mysql.stats_meta and mysql.stats_histograms;

Reply:
Updated.